blocks/leader-profiles/leader-profiles.css

```diff
@@ -0,0 +1,84 @@
+.leader-profiles>ul {
```

Comment:
Can you set `cursor: pointer` so its visually indicated that you can click the cards?